internal/common/function/function.go

```diff
@@ -95,10 +95,17 @@ func ValidateBackupSpec(ctx context.Context, clientInstance client.Client, oadpN
 			return fmt.Errorf("NonAdminBackup spec.backupSpec.includedNamespaces can not contain namespaces other than: %s", nonAdminBackup.Namespace)
 		}
 	}
-	if enforcedBackupSpec.IncludedNamespaces != nil {
-		if !containsOnlyNamespace(enforcedBackupSpec.IncludedNamespaces, nonAdminBackup.Namespace) {
-			return fmt.Errorf("NonAdminBackup spec.backupSpec.includedNamespaces enforced value by admin user violates NAC usage")
-		}
+
+	if nonAdminBackup.Spec.BackupSpec.ExcludedNamespaces != nil {
+		return fmt.Errorf("NonAdminBackup spec.backupSpec.excludedNamespaces is restricted")
+	}
+
+	if nonAdminBackup.Spec.BackupSpec.IncludeClusterResources != nil && *nonAdminBackup.Spec.BackupSpec.IncludeClusterResources {
+		return fmt.Errorf("NonAdminBackup spec.backupSpec.includeClusterResources can not be set or must be set to false")
+	}
+
+	if len(nonAdminBackup.Spec.BackupSpec.IncludedClusterScopedResources) > 0 {
+		return fmt.Errorf("NonAdminBackup spec.backupSpec.includedClusterScopedResources is restricted, only an empty list is allowed")
```

Comment:
Please make this error more informative. What happens when empty list is entered?

Similar error to the enforced admin?
```go
return false, fmt.Errorf("DPA spec.nonAdmin.includedClusterScopedResources is restricted and must remain empty; it cannot be specified")
```